tests/neg/i21215.scala

```diff
@@ -0,0 +1,44 @@
+
+trait FlatMap[F[_]]:
+  def flatMap[A, B](fa: F[A])(f: A => F[B]): F[B] = ???
+  def ifM[B](ifTrue: => F[B], ifFalse: => F[B])(implicit F: FlatMap[F]): F[B] = ???
+trait Monad[F[_]] extends FlatMap[F]
+trait MonadError[F[_], E] extends Monad[F]:
+  def raiseError[A](e: E): F[A]
+trait Temporal[F[_]] extends MonadError[F, Throwable]
+
+trait FlatMapOps[F[_], A]:
+  def flatMap[B](f: A => F[B]): F[B] = ???
```

Comment:
With additional map method we can make it a `pos` test
```suggestion
  def map[B](f: A => B): F[B] = ??? 
  def flatMap[B](f: A => F[B]): F[B] = ???
```